File: Old/Removed 2025/Removed 2025-03-25/ValidatorNumericProperty.cpp
#include "ValidatorNumericProperty.h"

namespace Alternet::UI
{
	ValidatorNumericProperty::ValidatorNumericProperty()
	{

	}

	ValidatorNumericProperty::~ValidatorNumericProperty()
	{
	}

	void ValidatorNumericProperty::DeleteValidatorNumericProperty(void* handle)
	{
		delete (wxNumericPropertyValidator*)handle;
	}

	void* ValidatorNumericProperty::CreateValidatorNumericProperty(int numericType, int valBase)
	{
		return new wxNumericPropertyValidator(
			(wxNumericPropertyValidator::NumericType)numericType, valBase);
	}
}

File: Old/Removed 2025/Removed 2025-01-16/NativeApi/Validator/ValidatorNumericProperty.cs
#pragma warning disable
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace NativeApi.Api
{
    //https://docs.wxwidgets.org/3.2/classwx_numeric_property_validator.html
    public class ValidatorNumericProperty : ValidatorText
    {
        public static void DeleteValidatorNumericProperty(IntPtr handle) { }
        public static IntPtr CreateValidatorNumericProperty(int numericType, int valBase) => default;
    }
}